### PR TITLE
Add password max_length argument

### PR DESCRIPTION
**Pull request by onigra**
faker has max_length argument in password method

https://github.com/stympy/faker#fakerinternet

### CODE REVIEW

**Comment by rocknruby** [comment]
@EmmanuelOga are you planning to accept this PR?

**Comment by EmmanuelOga** [comment]
Hey @rocknruby lately I haven't have any time to maintain this gem (#161). Would you like to help? I can make you admin of the repo and give you rubygems admin access. Thanks!

File: lib/ffaker/internet.rb [hunk 1]
```diff
@@ -74,12 +74,18 @@ def slug(words = nil, glue = nil)
...
-    def password(min_length=0)
+    def password(min_length = 8, max_length = 16)
       temp = Lorem.words.join
+
       while temp.length < min_length
         temp += Lorem.word
       end
-      return temp
+
+      unless max_length < min_length
```
Comment (rocknruby) [inline lib/ffaker/internet.rb]: @onigra, this should only be done if temp.length > max_length, right? 

e.g.
```ruby
temp.slice!(0, max_length) if temp.length > max_length && max_length > min_length
```

**Comment by rocknruby** [comment]
I change it by myself

**Comment by onigra** [comment]
@rocknruby oh, I'm sorry reply is delayed. thank you for merge and improve! :smiley:

**Comment by rocknruby** [comment]
you're welcome @onigra :octocat:

**Pull request merged**